README.md

```diff
@@ -1 +1,96 @@
-# Reloader
\ No newline at end of file
+# RELOADER
+
+## WHY NAME RELOADER
+
+In english language, Reloader is a thing/tool that can reload certain stuff. So refereig to that meaning relaoder can reload
+
+## Problem
+
+We would like to watch if some change happens in `ConfigMap` and `Secret` objects and then perform certain upgrade on relavent `Deployment`, `Deamonset` and `Statefulset`
+
+## Solution
+
+Reloader can watch any changes in `ConfigMap` and `Secret` objects and then performs rolling upgrades on their associated `Deployments`, `Deamonsets` and `Statefulsets` and updating these dynamically.  
```

Comment:
Update it to: 
Reloader can watch any changes in `ConfigMap` and `Secret` objects and update or recreate `Pods` for their associated `Deployments`, `Deamonsets` and `Statefulsets. In this way `Pods` can get the latest changes in `ConfigMap` or `Secret` objects.